baseapp-profiles/baseapp_profiles/graphql/mutations.py

```diff
@@ -114,6 +114,43 @@ def perform_mutate(cls, serializer, info):
         )
 
 
+RoleTypeEnum = graphene.Enum.from_enum(ProfileUserRole.ProfileRoles)
```

Comment:
instead of defining it here again do `from .object_types import ProfileRoleTypesEnum`